src/IECoreGL/Selector.cpp

```diff
@@ -396,14 +424,28 @@ class Selector::Implementation : public IECore::RefCounted
 				throw IECore::Exception( "ID shader does not have an ieCoreGLNameOut output" );
 			}
 
+			GLint depthDataLocation = 0;
+			if( m_useCameraDepth )
+			{
+				depthDataLocation = glGetFragDataLocation( shader->program(), "ieCoreGLCameraDepth" );
+				if( depthDataLocation < 0 )
```

Comment:
I wanted to do a similar test to check that the fragment shader is getting `geometryP`, but I wasn't able to find anything in OpenGL that would allow that. Perhaps I'm missing something?

Reply:
I don't know of one off the top of my head, but the check you have already seems sufficient to me.